lib/experimental/class-theme-json-webfonts-handler.php

```diff
@@ -0,0 +1,537 @@
+<?php
+/**
+ * Handler for theme.json webfonts.
+ *
+ * @package    WordPress
+ * @subpackage WebFonts
+ * @since      6.0.0
+ */
+
+if ( ! function_exists( '_wp_theme_json_webfonts_handler' ) ) {
+	add_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' );
+	/**
+	 * Runs the theme.json webfonts handler to expose defined webfonts
+	 * to the editor's typography pickers and generate `@font-face` style
```

Comment:
```suggestion
	 * to the editor's typography pickers and generate '@font-face' style
```
- Officially, markdown syntax is only valid in the body of a docblock, not the title.